crates/anvil/core/src/eth/wallet.rs

```diff
@@ -0,0 +1,75 @@
+use alloy_primitives::{map::HashMap, Address, ChainId};
+use serde::{Deserialize, Serialize};
+
+/// The capability to perform [EIP-7702][eip-7702] delegations, sponsored by the sequencer.
+///
+/// The sequencer will only perform delegations, and act on behalf of delegated accounts, if the
+/// account delegates to one of the addresses specified within this capability.
+///
+/// [eip-7702]: https://eips.ethereum.org/EIPS/eip-7702
+#[derive(Debug, Clone, Eq, PartialEq, Deserialize, Serialize)]
+pub struct DelegationCapability {
+    /// A list of valid delegation contracts.
+    pub addresses: Vec<Address>,
+}
```

Comment:
all of these should be moved to alloy, but can do that once that pr is merged and published